Improve example of contain-intrinsic-size with selection (#21428)
mdn/content: files/en-us/web/css/contain-intrinsic-size/index.md

#### JavaScript

> The code below adds and removes styles to the containing element based on the selected options.

Reviewer: ```suggestion
The code below adds styles to and removes styles from the containing element based on the selected options.
```

The HTML defines two buttons, a container element that is subject to containment via the `content-visibility` property.

```html

> <label for="contain_intrinsic_size_selector">contain-intrinsic-size:</label>
> <select id="contain_intrinsic_size_selector">
>   <option>none</option>
>   <option>40px 130px</option>
>   <option>auto 40px auto 130px</option>
> </select>
> 
> <label for="contain_selector">contain:</label>
> <select id="contain_selector">
>   <option>none</option>
>   <option>size</option>
>   <option>strict</option>
> </select>
> 
> <label for="content_visibility_selector">content-visibility:</label>
> <select id="content_visibility_selector">
>   <option>visible</option>
>   <option>auto</option>
>   <option>hidden</option>
> </select>
> 

Reviewer: ```suggestion
<p>
<label for="contain_intrinsic_size_selector">contain-intrinsic-size:</label>
<select id="contain_intrinsic_size_selector">
  <option>none</option>
  <option>40px 130px</option>
  <option>auto 40px auto 130px</option>
</select>;<br/>

<label for="contain_selector">contain:</label>
<select id="contain_selector">
  <option>none</option>
  <option>size</option>
  <option>strict</option>
</select>;<br/>

<label for="content_visibility_selector">content-visibility:</label>
<select id="content_visibility_selector">
  <option>visible</option>
  <option>auto</option>
  <option>hidden</option>
</select>;
</p>

```
This is obviously optional, but I think it will make the example easier to understand.

Author: Thanks!

<div id="contained_element">
  <div class="child_element"></div>
</div>
```

#### Result

> Change the selections to compare the results with different style settings.
> Note that when the content is visible or auto changing `contain-intrinsic-size` makes no difference.

Author: ```suggestion
Use the selectors to apply the given styles to the containing `div` element.
Note that when the `content-visibility` is `visible` or `auto`, changing `contain-intrinsic-size` makes no difference.
```

> Note that when the `content-visibility` is `visible` or `auto`, changing `contain-intrinsic-size` makes no difference.

Reviewer: ```suggestion
Note that when `content-visibility` is `visible` or `auto`, changing `contain-intrinsic-size` makes no difference.
```
